src/index.ts

```diff
@@ -2,11 +2,14 @@ let totalPersonas: number = Number(
   prompt(`Ingrese cantidad de personas en la disco`)
 );
 let personas: number[] = new Array(totalPersonas);
-let i: number;
 let menores21: number = 0;
 let mayores21: number = 0;
 
-for (i = 0; i < totalPersonas; i++) {
+function aleatorio(min: number, max: number): number {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+for (let i = 0; i < totalPersonas; i++) {
```

Comment:
Es una buena practica declarar el indice que usamos para el for, que en este caso es **i**, dentro del mismo for y no como una variable global. Por eso lo borramos de arriba y agregamos let antes del i del for.